jsonschema-module-jackson/src/main/java/com/github/victools/jsonschema/module/jackson/JacksonOption.java

```diff
@@ -63,5 +63,10 @@
     /**
      * Use this option to skip the transformation according to {@code @JsonTypeInfo} annotations (typically used to identify specific subtypes).
      */
-    IGNORE_TYPE_INFO_TRANSFORM;
+    IGNORE_TYPE_INFO_TRANSFORM,
+    /**
+     * Use this option to follow the required flag based on {@code @JsonProperty} required attribute
+     * By default, for backward compatibility, the "required" attribute of @JsonProperty is not
```

Comment:
checkstyle complains about a missing period symbol to end the first sentence.
The second part seems incomplete as well and isn't really necessary.
```suggestion
     * Use this option to include fields annotated with {@code @JsonProperty(required = true)} in the containing type's list of "required" properties.
```